spec/metrician_spec.rb

```diff
@@ -19,7 +19,7 @@
     end
 
     specify "per command instrumentation" do
-      Metrician.configuration[:database][:command] = true
+      Metrician.configuration[:database][:command][:enabled] = true
```

Comment:
I feel like there might be something odd/wrong going on with the database config. I just saw this bit and edited it to be inline with everything else.